File: solucoes/2021/nivel jr/fase 2/js/pangrama.js
var alfabeto = new Set('abcdefghijlmnopqrstuvxz')
var F

scanf("%s", "F")

var letras_encontradas = new Set()
for(var i = 0; i < F.length; i++){
    if (/[a-zA-Z]/.test(C)){
        letras_encontradas.add(C)
    }
}

if(letras_encontradas.length == alfabeto.length){
    printf("%s", "S")
}
else{
    printf("%s", "N")
}

File: solucoes/2021/nivel jr/fase 2/py/pangrama.py
alfabeto = set('abcdefghijlmnopqrstuvxz')
F = input()

letras_encontradas = set()
for C in F:
    if C.isalpha():
        letras_encontradas.add(C)

if len(letras_encontradas) == len(alfabeto):
    print("S")
else:
    print("N")
